File: src/main/scala/work/chiro/game/bullet/HeroBullet.scala
package work.chiro.game.bullet

import work.chiro.game.animate.AnimateContainer
import work.chiro.game.application.ImageResourceFactory
import work.chiro.game.basic.ScaleType.Position

/**
 * @author chiro2001
 */
class HeroBullet(posInit: Position, animateContainer: AnimateContainer, power: Int)
  extends AbstractBullet(posInit, animateContainer, power) {
  override def getImage = HeroBullet.getImage
}

object HeroBullet extends ImageResourceFactory {
  override def getImageCachedPath = "images/bullet_hero.png"
}

File: app/src/main/java/work/chiro/game/bullet/HeroBullet.java
package work.chiro.game.bullet;

import work.chiro.game.animate.AnimateContainer;
import work.chiro.game.config.AbstractConfig;
import work.chiro.game.vector.Vec2;

/**
 * @author hitsz
 */
public class HeroBullet extends BaseBullet {
    public HeroBullet(AbstractConfig config, Vec2 posInit, AnimateContainer animateContainer, int power) {
        super(config, posInit, animateContainer, power);
    }
}
